static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -91,7 +91,7 @@ function UserActionsNavigator({event, crumbs}: Props) {
                   color={item.color}
                   description={item.description}
                 />
-                <ActionCategory category={item} />
+                <ActionCategory action={item} />
```

Comment:
What is the type of `item` at this line?

Reply:
it has the type `Crumb`.